mk/spksrc.service.installer

```diff
@@ -1,12 +1,16 @@
 #!/bin/sh
```

Comment:
Should this file get a dsm6 postfix (for consistency) `spksrc.service.installer.dsm6` or can dsm6/7 share this file now? How much dsm 7 specific stuff is left anyway?

Reply:
✔️ 
current differences are:
- different path of FWPORTS_FILE
- save INST_VARIABLES is implemented in DSM6 only
- several syno_user/group functions are not (yet?) updated for DSM7
- DSM6 creates the folder `${SYNOPKG_PKGVAR}` and sets permissions
- DSM6 still creates the /usr/local/{package} folder in postinst and removes it in postuninst
- DSM7 has the default uninstall-wizard with the option to delete permanent storage data
- DSM6 removes the package user and group in postuninst
- DSM6 saves and restores the var folder between preupgrade and postupgrade
-

Reply:
Thanks, yea so sill a lot, Best keep them separate then :+1: 
Regarding syno_user/group functions I don't think they can ever be ported due to lower permissions. I'm fine with it since the UI from the DSM now has much better/fine-grained permissions through the "internal users"